ibm/resource_ibm_database.go

```diff
@@ -607,7 +608,20 @@ func resourceIBMDatabaseInstanceRead(d *schema.ResourceData, meta interface{}) e
 		return nil
 	}
 
-	d.Set("tags", instance.Tags)
```

Comment:
Can we move get tags also to separate function like update..so we can use in multiple resources in future

Reply:
Moved to GetTags in structures.go